### ISSUE TITLE
Create App logo

### ISSUE THREAD

**Issue by sand4rbh** [sanjana1903/Whizzard#1]
@sanjana1903  We need an app logo for the app drawer and also for Splash Screen.
Can you start working on it or assign someone else who can?

**Comment by sanjana1903** [comment]
@SandarbhS yes I will start working on it and send it to you when it's done
